ibis/backends/datafusion/tests/conftest.py

```diff
@@ -0,0 +1,66 @@
+from pathlib import Path
+
+import pandas as pd
+import pyarrow as pa
+
+import ibis
+import ibis.expr.types as ir
+import ibis.expr.operations as ops
+from ibis.backends.tests.base import BackendTest, RoundHalfToEven, RoundAwayFromZero
+
+
+class TestConf(BackendTest, RoundAwayFromZero):
+    # check_names = False
+    # additional_skipped_operations = frozenset({ops.StringSQLLike})
+    # supported_to_timestamp_units = BackendTest.supported_to_timestamp_units | {
+    #     'ns'
+    # }
+    # supports_divide_by_zero = True
+    # returned_timestamp_unit = 'ns'
+    bool_is_int = True
+
+    @staticmethod
+    def connect(data_directory: Path):
+        # can be various types:
+        #   pyarrow.RecordBatch
+        #   parquet file path
+        #   csv file path
+        client = ibis.datafusion.connect({})
+        client.register_csv(
+            name='functional_alltypes',
+            path=data_directory / 'functional_alltypes.csv',
+            schema=pa.schema([
+                ('index', 'int64'),
+                ('Unnamed 0', 'int64'),
+                ('id', 'int64'),
+                ('bool_col', 'int8'),
+                ('tinyint_col', 'int8'),
+                ('smallint_col', 'int16'),
+                ('int_col', 'int32'),
+                ('bigint_col', 'int64'),
+                ('float_col', 'float32'),
+                ('double_col', 'float64'),
+                ('date_string_col', 'string'),
+                ('string_col', 'string'),
+                ('timestamp_col', 'string'),
+                ('year', 'int64'),
+                ('month', 'int64'),
+            ])
+        )
+        client.register_csv(
+            name='batting',
```

Comment:
I believe there are also parquet equivalents here, maybe we should use parquet for at least one of these datasets?

Reply:
Definitely!